test/albumslist.test.js

```diff
@@ -3,38 +3,34 @@ const request = require('supertest'),
   { factory } = require('factory-girl'),
   { hashPassword } = require('../app/helpers/hasher'),
   User = require('../app/models').user,
-  { getAlbumById } = require('../app/services/typicode'),
+  { albumsListMock } = require('./support/mocking'),
   correctEmail = 'albumslist@wolox.com.ar',
   correctEmail2 = 'albumlists2@wolox.com.ar',
   correctPassword = 'password',
   validationErrorStatus = 401,
   permissionErrorStatus = 403,
   tokenErrorStatus = 500;
 
-const buyTwoAlbums = (id1, id2, buyerToken) =>
-  request(app)
+const buyTwoAlbums = (id1, id2, buyerToken) => {
+  albumsListMock(1);
+  albumsListMock(10);
+  return request(app)
     .post(`/albums/${id1}`)
     .send({ token: buyerToken })
     .then(() =>
       request(app)
         .post(`/albums/${id2}`)
         .send({ token: buyerToken })
     );
+};
 
 const checkBoughtAlbums = (response, id1, id2) => {
   expect(response.status).toBe(200);
   expect(response.body.length).toBe(2);
   expect(response.body[0].albumId).toBe(id1);
   expect(response.body[1].albumId).toBe(id2);
-  return getAlbumById(id1)
-    .then(album => {
-      expect(response.body[0].albumName).toBe(album.title);
-    })
-    .then(() =>
-      getAlbumById(id2).then(album => {
-        expect(response.body[1].albumName).toBe(album.title);
-      })
-    );
+  expect(response.body[0].albumName).toBe('quidem molestiae enim');
+  expect(response.body[1].albumName).toBe('quidem molestiae enim');
```

Comment:
Lets try to avoid bare strings like this. If the mock ever changes then you'd have to manually change these two. There are two options you can take: 
* Defining `quidem molestiae enim` in an exports and resusing it whenever
* Passing `quidem molestiae enim` as an aditional param in the mock function.